library/core/src/macros/mod.rs

```diff
@@ -334,15 +334,15 @@ macro_rules! r#try {
 /// formatted according to the specified format string and the result will be passed to the writer.
 /// The writer may be any value with a `write_fmt` method; generally this comes from an
 /// implementation of either the [`std::fmt::Write`] or the [`std::io::Write`] trait. The macro
```

Comment:
What about this to match the change below?

```suggestion
/// implementation of either the [`fmt::Write`] or the [`io::Write`] trait. The macro
```

Reply:
Sounds good to me.